### PR TITLE
Feat Convert Homepage buttons to shadcn/ui

### PR DESCRIPTION
**Pull request by Cormwww**
Hello, sorry about the confusion with the other PR but I think that this should now work. I added Shadcn/ui to the repository and added the buttons to the page.tsx file in the repository please tell me if there are ant issues with it. Thanks!

### CODE REVIEW

**Comment by nine-codes** [comment]
Ovo treba zatvoriti @alempasalic @bambooch

**Comment by bambooch** [comment]
Hey @Cormwww I've just tested your fork and it seems it doesn't work, see screenshot below, this is after running `pnpm turbo install` and `pnpm turbo dev` from the root directory 

also I believe that shadcn was correctly configured as part of #79 please take a look at that and at PRs such as #85 to see how to correctly add shadcn components to the repo

image

File: frontend/src/app/page.tsx [hunk 1]
```diff
@@ -1,441 +1,131 @@
...
   useEffect(() => {
-    const today = (new Date().getDay() + 6) % 7;
-    setTempDepartureDate(today);
-
-    const fetchRoutes = async () => {
+    const fetchAgency = async () => {
       try {
-        const fetchedRoutes = await RouteService.getRoutes();
-        setOriginalRoutes(fetchedRoutes);
+        const fetchedAgency = await AgencyService.getAgency(id as string);
```
Comment (bambooch) [inline frontend/src/app/page.tsx:21]: you seem to be adding this agency/city functionality as part of your PRs that should just change the buttons on the homepage. please make sure you understand the requirements before opening any PRs

**Review by bambooch — CHANGES_REQUESTED** [review]
Too much code has been deleted here for a button switch. Once more, please understand the requirements and then write the code. If you need clarification, please let me know what is confusing you.

**Comment by bambooch** [comment]
@Cormwww I see that you pushed some commit in the last hour but did not address the comments above. Please consider them and let us know if you need support.

**Pull request closed without merging**